src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,203 @@
+  "jeffposnick": 'Jeff Posnick é membro da equipe de Relações com desenvolvedores da Web do Google, com sede em Nova York. Seu foco está em <a href="https://developers.google.com/web/tools/workbox/">Workbox</a>, um conjunto de bibliotecas trabalhador de serviço para Progressive Web Apps. Ele bloga em <a href="https://jeffy.info">https://jeffy.info</a> e tweeta como <a href="https://twitter.com/jeffposnick">@jeffposnick</a>.',
+  "khempenius": 'Katie Hempenius é engenheira da equipe do Chrome, onde trabalha para tornar a web mais rápida.',
+  "AVGP": 'Martin Splitt é um defensor de desenvolvedores da equipe de ecossistemas da Web no Google, onde trabalha para manter a Web detectável.',
+  "nektarios-paisios": 'Nektarios Paisios é um engenheiro de software que trabalha com acessibilidade do Chrome nos últimos 5 anos. Ele se concentra principalmente em tornar o Chrome compatível com software auxiliar de terceiros, como leitores e ampliadores de tela. Antes de trabalhar na acessibilidade do Chrome, Nektarios trabalhou em várias outras funções na empresa, como acessibilidade do GSuite e anúncios gráficos. Nektarios é Ph.D. em Ciência da Computação pela New York University.',
+  "patrickhulce": 'Patrick Hulce é um ex-engenheiro Chrome, fundador do <a href="https://eris.ventures/">Eris Ventures</a>, membro da equipe do núcleo de <a href="https://github.com/GoogleChrome/lighthouse">Lighthouse</a> e <a href="https://github.com/GoogleChrome/lighthouse-ci">Lighthouse CI</a>, co-organizador do <a href="https://www.meetup.com/DallasJS/">DallasJS</a> meetup, e autor do projeto <a href="https://github.com/patrickhulce/third-party-web">third-party-web</a>.',
+  "paulcalvano": 'Paul Calvano é um Arquiteto de Desempenho Web em <a href="https://www.akamai.com/">Akamai</a>, onde ele ajuda as empresas a melhorar o desempenho de seus sites. Ele também é co-mantenedor do projeto HTTP Archive. Você pode encontrá-lo twittar em <a href="https://twitter.com/paulcalvano">@paulcalvano</a>, o blog em <a href="https://paulcalvano.com">http://paulcalvano.com</a> E compartilhamento de pesquisa HTTP Arquivo em <a href="https://discuss.httparchive.org">https://discuss.httparchive.org</a>.',
+  "rachellcostello": 'Rachel Costello é uma SEO técnica &amp; Content Manager em <a href="https://www.deepcrawl.com/">DeepCrawl</a> um conferencista internacional que gasta seu tempo pesquisando e comunicar os mais recentes desenvolvimentos em pesquisa e. Rachel atualmente administra a produção de <a href="https://www.deepcrawl.com/knowledge/white-papers/">technical SEO white papers</a> e peças de pesquisa para DeepCrawl, e é um colunista regular para <a href="https://www.searchenginejournal.com/author/rachel-costello/">Search Engine Journal</a>.',
```

Comment:
Should we translate technical SEO white papers? Not sure if there is an equivalent phrase in Portuguese and if you use the term "white paper"?

Reply:
> Should we translate technical SEO white papers? Not sure if there is an equivalent phrase in Portuguese and if you use the term "white paper"?

I can't get a legal term to describe "technical SEO white papers", oh unless some other translator with better interpretation skills finds a more coherent term